content/docs/command-reference/diff.md

```diff
@@ -44,18 +44,13 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Options
 
-- `--targets <paths>` - limit command scope to these paths. When specifying
-  arguments for `--targets` before `a_rev`/`b_rev`, you should use `--` after
-  this option's arguments, e.g.:
+- `--targets <paths>` - limit command scope to these paths.
 
-  ```dvc
```

Comment:
same here - why removing this? We are complicating users life this way for the sake of preserving "perfect" and clean structure to my mind.